lib/generators/shopify_app/install/templates/_flash_messages.html.erb

```diff
@@ -1,13 +0,0 @@
-<script type="text/javascript">
-  var eventName = typeof(Turbolinks) !== 'undefined' ? 'page:change' : 'DOMContentLoaded';
-
-  document.addEventListener(eventName, function() {
-    <% if flash[:notice] %>
-      ShopifyApp.flashNotice("<%= j flash[:notice].html_safe %>");
-    <% end %>
-
-    <% if flash[:error] %>
-      ShopifyApp.flashError("<%= j flash[:error].html_safe %>");
-    <% end %>
-  });
-</script>
```

Comment:
I'm not sure about removing this as its required for ESDK flash message to work. How are you showing your flash message errors?